tests_e2e/tests/scripts/agent_cgroups_process_check-unknown_process_check.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env pypy3
+# Microsoft Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script forces the process check by putting unknown process in the agent's cgroup
+
+import os
+import subprocess
+import datetime
+
+from assertpy import fail
+
+from azurelinuxagent.common.utils import shellutil
+from tests_e2e.tests.lib.cgroup_helpers import get_agent_cgroup_mount_path, \
+    BASE_CGROUP, check_agent_quota_disabled, check_log_message
+from tests_e2e.tests.lib.logging import log
+from tests_e2e.tests.lib.retry import retry_if_false
+
+
+def prepare_agent():
+    check_time = datetime.datetime.utcnow()
+    log.info("Executing script update-waagent-conf to enable agent cgroups config flag")
+    result = shellutil.run_command(["update-waagent-conf", "Debug.CgroupCheckPeriod=20", "Debug.CgroupLogMetrics=y",
+                                    "Debug.CgroupDisableOnProcessCheckFailure=y",
+                                    "Debug.CgroupDisableOnQuotaCheckFailure=n"])
+    log.info("Successfully enabled agent cgroups config flag: {0}".format(result))
+
+    found: bool = retry_if_false(lambda: check_log_message(" Agent cgroups enabled: True", after_timestamp=check_time))
+    if not found:
+        fail("Agent cgroups not enabled")
+
+
+def creating_dummy_process():
+    log.info("Creating dummy process to add to agent's cgroup")
+    dd_command = ["dd", "if=/dev/zero", "of=/dev/null"]
```

Comment:
what is the intention of using dd? this will consume high CPU. sleep may be good enough

Reply:
no intention, I remembered to use this command, decided to use. I can change it